lib/arch_def.rb

```diff
@@ -352,6 +354,21 @@ def implemented_extensions
     @implemented_extensions
   end
 
+  # @return [Array<ExtensionRequirement>] List of extensions that are explicitly required by an arch def
+  def mandatory_extensions
+    raise "mandatory_extensions is only valid for a partially configured defintion" unless partially_configured?
+
+    return @mandatory_extensions unless @mandatory_extensions.nil?
+
+    @mandatory_extensions = []
+    if @arch_def.key?("mandtory_extensions")
```

Comment:
Typo: mandtory

Reply:
thanks.

Reply:
I didn't test it; I see now that it's not obvious at all, but this PR is a "draft"